packages/puppeteer-extra-plugin-stealth/evasions/navigator.hardwareConcurrency/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const {PuppeteerExtraPlugin} = require('puppeteer-extra-plugin')
+
+/**
+ * Allows faking CPU core number. This allows creating different profiles.
+ * Will overwrite `navigator.hardwareConcurrency` property.
+ */
+class Plugin extends PuppeteerExtraPlugin {
+    constructor(opts = {}) {
+        super(opts)
+    }
+
+    get name() {
+        return 'stealth/evasions/navigator.hardwareConcurrency'
+    }
+
+
+    async onPageCreated(page) {
+    let cores = this.opts.cores || false;
+        await page.evaluateOnNewDocument((cores) => {
+            let originalCores = navigator.hardwareConcurrency;
+             cores = cores || originalCores;
```

Comment:
This line seems to be doing nothing, since you're doing the same in the assignment below

Reply:
Indeed. An oversight, should I add a commit ?

Reply:
Sure 👍🏻